infrastructure/README.md

```diff
@@ -0,0 +1,13 @@
+# Setting up local developer testnet
```

Comment:
Please update this readme.
User can start network using make command
which is present in makefile in this folder